### ISSUE TITLE
People are using Demo for real applications by mistake

### ISSUE THREAD

**Issue by lippytak** [codeforamerica/clean#488]
Let's fix it! One idea is a prominent flash message on the demo instance. What are other solutions?

**Comment by lippytak** [comment]
I think one person, two cases. I sent both cases to RM so they should be okay.

**Comment by alanjosephwilliams** [comment]
Thanks for taking care of the cases so far. We should email Cindy as
well—or perhaps the entire food bank team.

Let's change the landing page to reflect that it is the demo version of the
site. That probably looks less like a modal or flashed message, and more
like a regular part of the view.

On Thu, Apr 16, 2015 at 8:13 PM, Jake Solomon <notifications@github.com>
wrote:

> I think one person, two cases. I sent both cases to RM so they should be
> okay.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/codeforamerica/clean/issues/488#issuecomment-93879113>
> .
>



-- 
t:  @alanjosephwilli
p: 817 713 6264

**Comment by lippytak** [comment]
Happened again today!

**Comment by alanjosephwilliams** [comment]
It would be nice to sneak this into this week's sprint—@daguar let's touch base about the implications of maintaining separate markup and styles for demo. 

We could write the markup as a partial that contains its styles inline—perhaps can we conditionally call it?

**Comment by lippytak** [comment]
I think we should be working to move folks off Clean > getcalfresh.org asap
so hesitate to invest in this.

**Comment by alanjosephwilliams** [comment]
I agree completely, but feel like this falls into a category of acceptable
ongoing maintenance—if and only if the scope of maintaining not-identical
codebases is reasonably small.

On Mon, May 18, 2015 at 10:31 AM, Jake Solomon <notifications@github.com>
wrote:

> I think we should be working to move folks off Clean > getcalfresh.org
> asap
> so hesitate to invest in this.
>
> On Mon, May 18, 2015 at 10:29 AM, Alan Joseph Williams <
> notifications@github.com> wrote:
>
> > It would be nice to sneak this into this week's sprint—@daguar
> > <https://github.com/daguar> let's touch base about the implications of
> > maintaining separate markup and styles for demo.
> >
> > We could write the markup as a partial that contains its styles
> > inline—perhaps can we conditionally call it?
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <
> https://github.com/codeforamerica/clean/issues/488#issuecomment-103142611>
>
> > .
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/codeforamerica/clean/issues/488#issuecomment-103143016>
> .
>



-- 
t:  @alanjosephwilli
p: 817 713 6264

**Comment by daguar** [comment]
This is super small so I'd say we do it.

Alan — if you do a simple div that can go in the layout that just says something like "this is the Clean demo, to submit a real app, click here" or something then I can easily make it conditional on an env var and we're done.

**Comment by alanjosephwilliams** [comment]
Perfect :+1:

On Mon, May 18, 2015 at 10:40 AM, Dave Guarino <notifications@github.com>
wrote:

> This is super small so I'd say we do it.
>
> Alan — if you do a simple div that can go in the layout that just says
> something like "this is the Clean demo, to submit a real app, click here"
> or something then I can easily make it conditional on an env var and we're
> done.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/codeforamerica/clean/issues/488#issuecomment-103145148>
> .
>



-- 
t:  @alanjosephwilli
p: 817 713 6264

**Comment by daguar** [comment]
Pinged the FB with correct URLs just as a reminder
